cql3/statements/select_statement.cc

```diff
@@ -1734,6 +1736,21 @@ std::unique_ptr<prepared_statement> select_statement::prepare(data_dictionary::d
             stats,
             std::move(prepared_attrs)
         );
+    } else if (db.get_config().check_experimental(db::experimental_features_t::feature::BROADCAST_TABLES) &&
+               raft::broadcast_tables::is_broadcast_table_statement(keyspace(), column_family())) [[unlikely]] {
```

Comment:
Again the experimental check and unlikely can be dropped.

Reply:
Done.